src/test/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationServiceTest.java

```diff
@@ -106,7 +109,35 @@ void shouldSendNotificationEmail_whenCaseDataIsProvided() {
 
     @Test
     void shouldThrowNullPointerException_whenCaseDataIsNull() {
+
         assertThrows(NullPointerException.class, () ->
-            service.notifyRobotics(null));
+            service.notifyRobotics(null, true));
+    }
+
+    @Test
+    @SneakyThrows
+    void shouldSendNotificationEmailForMultiParty_whenCaseDataIsProvided() {
+        CaseData caseData = CaseDataBuilder.builder().atStateClaimDetailsNotified().build().toBuilder()
+            .respondent2(PartyBuilder.builder().individual().build())
+            .addRespondent2(YES)
+            .respondent2SameLegalRepresentative(NO)
+            .build();
+
+        boolean multiPartyScenario = isMultiPartyScenario(caseData);
+        service.notifyRobotics(caseData, multiPartyScenario);
+
+        verify(sendGridClient).sendEmail(eq(emailConfiguration.getSender()), emailDataArgumentCaptor.capture());
+
+        EmailData capturedEmailData = emailDataArgumentCaptor.getValue();
+        String reference = caseData.getLegacyCaseReference();
+        String message = format(
+            "Mulitparty claim data JSON is attached for %s",
```

Comment:
I dont think we are attaching the JSON payload for these emails anymore. Im not sure what the content should be of the message - doesnt mention anything specific on the ticket